app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from typing import Any
+from .errors import (NotVaccinatedError,
+                     OutdatedVaccineError,
+                     NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> Any:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
```

Comment:
The `NotVaccinatedError` should be raised with a descriptive message. According to the checklist, you should provide a message when raising exceptions. For example: `raise NotVaccinatedError('Visitor is not vaccinated.')`.